src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -78,6 +78,10 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
   plan$ = new BehaviorSubject<Plan | null>(null);
 
   formGroups: FormGroup[];
+  nameFormGroup: FormGroup<any>;
```

Comment:
I like that we are naming the form groups, but Im not sure this is the way to go.
I was thinking that instead of adding this additional properties, work on the form group initialization.
I've made this example https://stackblitz.com/edit/angular-formbuilder-bmo9gw?file=src%2Fapp%2Fapp.component.ts let me know what you think!

Reply:
Ooh I like this better — ty for ur Angular wisdom! Adding this now

Reply:
This might be more of a cleanup / housekeeping task that we might want to tackle when we have more time, so feel free to add a todo or a story in JIRA to follow up!

Reply:
Added a todo!